playwright/components/card/index.ts

```diff
@@ -0,0 +1,19 @@
+import type { Page } from "@playwright/test";
+import CARD from "./locators";
+
+// component preview locators
+export const card = (page: Page) => page.locator(CARD).nth(0);
+
+export const draggableCard = (page: Page, index: number) =>
+  page
+    .locator(`[data-element="draggable-card-${index}"]`)
+    .locator('[data-element="drag"]');
+
+export const draggableContainer = (page: Page, index: number) =>
+  page.locator(`[data-element="draggable-container-${index}"]`);
+
+export const columnCard = (page: Page) =>
+  page.locator(`[data-element="card-column"]`).locator("div:nth-child(2)");
+
+export const footerCard = (page: Page) =>
+  page.locator(`[data-element="card-footer"]`);
```

Comment:
Should normal single quotes be needed here rather than backticks?

Reply:
Same answer as previous. Yeah it should be single quotes. I had issues here and I experimented with passing an ${index} in the page.locator hence the need for back ticks. I'll get them changed to single quotes 👍🏻